### PR TITLE
do not index the login page for search engines

### PR DESCRIPTION
**Pull request by qw3ry**
... except it is the root page

### CODE REVIEW

**Comment by qw3ry** [comment]
Depends on https://github.com/scireum/sirius-web/pull/800

**Review by sabieber — APPROVED** [review]

File: src/main/resources/default/templates/biz/login.html.pasta [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <i:arg type="String" name="originalUrl"/>
+<i:arg type="boolean" name="noindex"
+       default="@('/' != CallContext.getCurrent().get(WebContext.class).getRawRequestedURI())"/>
```
Comment (andyHa) [inline src/main/resources/default/templates/biz/login.html.pasta:3]: is this also true for "http://host.tld"

**Review by andyHa — COMMENTED** [review]
See question inline.

**Review by meg1502 — APPROVED** [review]

File: src/main/resources/default/templates/biz/login.html.pasta (hunk 1 above)
Comment (qw3ry) [reply src/main/resources/default/templates/biz/login.html.pasta:3]: At least in S2 the URI is `http://host.tld`, while the `WebContext` gives me `/`

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/default/templates/biz/login.html.pasta (hunk 1 above)
Comment (andyHa) [reply src/main/resources/default/templates/biz/login.html.pasta:3]: +1

**Review by andyHa — APPROVED** [review]

**Pull request merged**